src/components/Questionnaire/QuestionnaireEditor.tsx

```diff
@@ -996,8 +1031,8 @@ export default function QuestionnaireEditor({ id }: QuestionnaireEditorProps) {
                   <div>
                     <CardTitle>
                       <p className="text-sm text-gray-700 font-medium mt-1">
-                        {questionnaire.questions?.length || 0} Question
-                        {questionnaire.questions?.length !== 1 ? "s" : ""}
+                        {questionnaire.questions?.length || 0} {t("question")}
+                        {questionnaire.questions?.length !== 1 ? t("s") : ""}
```

Comment:
Ah, I see...update this accordingly, similar to questionnaire I mentioned above.

https://www.i18next.com/translation-function/plurals